StreetDrop/StreetDrop/Data/Repositories/Setting/DefaultSettingsRepository.swift

```diff
@@ -87,4 +87,23 @@ final class DefaultSettingsRepository: SettingsRepository {
             )
         ]
     }
+    
+    func fetchLastSeenNoticeIdFromLocal() -> Single<Int?> {
+        Single.create { [weak self] observer in
+            if let noticeId = self?.myInfoStorage.fetchLastSeenNoticeId() {
+                observer(.success(noticeId))
+            } else {
+                observer(.success(nil))
+            }
```

Comment:
리턴 타입이 옵셔널이라면 가독성을 위해 아래 코드와 같이 해도 좋지 않을까 합니다!

```suggestion
            let noticeId = self?.myInfoStorage.fetchLastSeenNoticeId()
            observer(.success(noticeId))
```

Reply:
오오..! 그러네요!!